docs/root/intro/arch_overview/upstream/load_balancing/load_balancers.rst

```diff
@@ -132,16 +132,18 @@ host B has a weight of 2, then host A will have 21,846 entries and host B will h
 regardless of the configured host and locality weights, so in some extreme cases the actual
 proportions may differ from the configured weights. For example, if the total number of hosts is
 larger than the fixed table size, then some hosts will get 1 entry each and the rest will get 0,
-regardless of weight. Best practice is to monitor the :ref:`min_entries_per_host and
-max_entries_per_host gauges <config_cluster_manager_cluster_stats_maglev_lb>` to ensure no hosts
-are underrepresented or missing.
```

Comment:
Sorry I don't think it makes sense to move this part, as this is related to ring population, not the section below on ring stability.

In general I'm really not sure this PR makes anything that much clearer but if after moving this part back you still want an edit below feel free.

/wait

Reply:
I agree this does not makes sense to move that part below. Not sure why I moved that. I wanted to refer table_size config and let users set it.

Given that my original intention of clarifying whether Maglev performs  better or worse (that was the confusion I was having) when hosts change is not needed any more, I will close this.

Reply:
@mattklein123 Sorry. Now I remember - I wanted to refer table_size here for the folks who wanted to reduce the amount of disruption using MagLev. Can you please see if this change makes sense? Sorry for the run around.